fhirzeug/generators/python_pydantic/templates/fhirdate.py

```diff
@@ -1 +1,21 @@
-FHIRDate = str
+import pydantic
+
```

Comment:
I would say `fhirdate.py` is not really the name you want to use here?

Reply:
renamed the file!